include/style/widget.styl

```diff
@@ -26,5 +26,10 @@
             background: $light-grey
             color: $white-invert
 
+    &#toc
+        .card-content
+            max-height: 94vh
```

Comment:
Does this only work when ToC is the first widget in its sidebar? What if there is some other widgets above it? My suggestion is to leave out this part for now.

Reply:
pretty fair. done.